llms/mlx_lm/lora.py

```diff
@@ -160,10 +160,7 @@ def load_dataset(args):
     return train, valid, test
 
 
-if __name__ == "__main__":
-    parser = build_parser()
-    args = parser.parse_args()
-
+def run_lora(args, training_callback: TrainingCallback = None):
```

Comment:
Ok naming this `run` instead of `run_lora`? I think it will be slightly more flexible if we ever have multiple tuning strategies, and it's already clear it's for lora since it will be `lora.run`.